---
title: "Midterm 2 review"
---

# Chromatin structure

The DNA in eukaryotes is very long, around 2 meters in humans! Chromatin must
tightly pack all of that DNA inside the cell while still being dynamic enough to
facilitate transcriptional regulation.

## Core Histones

Core histones provide much greater compaction when compared to the equivalent
structures in bacteria. They are very highly conserved across species when
compared to linker histones.

The globular domain of the core histones have many charged amino acid near the
surface. Non-polar amino acids form the interior of the globular domain. Core
histones have tails on the N-terminal between 20 and 30 amino acids long. These
tails are disordered or unstructured and contain many lysine and arginine
residues. The tails are highly conserved, indicating some important function.

Chaperone chromatin assembly factors catalyze core histone assembly in the S
phase. Histone tails are very positively charged making them interact with the
negatively charged DNA backbone. A core nucleosome consists of 8 histone
proteins: H2A, H2B, H3 and H4.

## linker histones

A linker histone is an H1 protein and links groups of core histones together.
Linker histones also stabilize nucleosomes when they are in a compact
conformation.

Linker histones have a tripartite structure consisting of an N-terminal tail, a
globular domain, and a C-terminal region which may play a role in protein
stability.

## Heterochromatin

Heterochromatin is the inactive form of chromatin and is typically inaccessible
to transcription factors. Usually, heterochromatin is composed of highly
repetitive DNA sequences.

## Euchromatin

Euchromatin is condensed only at metaphase, genes may be active or inactive.
Inactive chromatin may be referred to as silent chromatin. Euchromatin is
usually modified via acetylation.

## Pioneer TFs

The cell needs some way to open closed chromatin. It does so using proteins
called pioneer transcription factors. FOXA and GATA are two examples. Pioneer
transcription factors have a high affinity to nucleosomes and a slow nuclear
mobility.

## Histone code hypothesis

The histone code refers to epigenetic modifications like methylation and
acetylation.

# Gene regulation

Gene regulation allows cells to change their phenotype based on their
environment. This allows cells to differentiate into different cell types.

## Gene profiling techniques

1. Northern blot is a classic technique that was able to detect a single
   transcript
2. qPCR is a technique first converts the mRNA to a cDNA library before
   performing quantitative PQR. Quantitative PCR measures the rate of
   replication of cDNA transcripts, allowing for quantification of relative
   expression levels.
3. Microarray tiles sequences to a known set of genes
4. RNA-sequencing first converts mRNA to cDNA then directly sequences the cDNA,
   no reference necessary.

## Major control points in gene regulation
1. Transcription
2. RNA processing
3. Translation
4. Post-translational modifications

